tests/main/interfaces-wayland/task.yaml

```diff
@@ -0,0 +1,65 @@
+summary: Ensure that the wayland interface works
+
+# Only test on classic Ubuntu amd64 systems that have wayland
+systems: [ ubuntu-1*-*64 ]
+
+prepare: |
+    . $TESTSLIB/pkgdb.sh
+    snap install --edge test-snapd-wayland
```

Comment:
One more thing - is test-snapd-wayland packaged somewhere? If there is a snapcraft.yaml for it, we should add it to `tests/lib/snaps` or if not add a comment where the git tree for this is.

Reply:
Yes, lp:~jdstrand/+git/test-snapd-wayland. I listed it in the description of the snap:
```
$ snap info test-snapd-wayland
name:      test-snapd-wayland
summary:   "Basic wayland test snap"
publisher: jdstrand
description: |
  Basic wayland test snap
  lp:~jdstrand/+git/test-snapd-wayland
...
```